### ISSUE TITLE
Transformer_handler_generalized.py needs some adjustment after some update

### ISSUE THREAD

**Issue by Makos63** [pytorch/serve#1044]
## Context

* torchserve version: 0.3.1
* torch-model-archiver version: 0.3.1
* torch version: 1.7.1
* torchvision version [if any]:
* torchtext version [if any]:
* torchaudio version [if any]:
* java version: 11.0.10" 2021-01-19
* Operating System and version: 	Ubuntu 18.04.5 LTS

## Your Environment

* Installed using source? [yes/no]: no
* Are you planning to deploy it using docker container? [yes/no]: no
* Is it a CPU or GPU environment?: GPU
* Using a default/custom handler? [If possible upload/share custom handler/model]:
* https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/Transformer_handler_generalized.py
* What kind of model is it e.g. vision, text, audio?: text
* Are you planning to use local models from model-store or public url being used e.g. from S3 bucket etc.?
  [If public url then provide link.]: local model
* Provide config.properties, logs [ts.log] and parameters used for model registration/update APIs: 
config.properties
inference_address=http://ip:8080
management_address=http://ip:8081
metrics_address=http://ip:8082
* Link to your project [if any]:

## Expected Behavior
Provided example should work after following the instruction
## Current Behavior
The default handler for question answering provided in Transformers example did not work. It failed in inference function while  reinterpreting input_batch with code answer_start_scores, answer_end_scores = self.model(input_batch).
Important to keep in mind, the error which shows in log says it is a bit further in the code, but the main issue comes from this line 180 (https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/Transformer_handler_generalized.py#L180)
## Possible Solution
Found the solution on StackOverflow which said that self.model(input_batch) should contain one more parameter self.model(input_batch,return_dict=False). More details in: https://stackoverflow.com/questions/64901831/huggingface-transformer-model-returns-string-instead-of-logits

I think it would be great if new code would be developed or at least the change in library be documented in readme.

## Steps to Reproduce
1. generate a model with pytroch-model-archiver and an example handler from https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/Transformer_handler_generalized.py
2. deploy it
3. send curl request 
4. will return internal prediction error
...

**Comment by HamidShojanazeri** [comment]
@Makos63 Thanks for reporting this, would you please share the version of Transformers, there is PR under development for supporting latest Transformers/ covering the changes in the API for Q&A.

**Comment by Makos63** [comment]
@HamidShojanazeri 

Running:

import transformers 
print(transformers.__version__)

gave me: 4.1.1

**Comment by msaroufim** [comment]
We are updating the example #1063 here

**Comment by HamidShojanazeri** [comment]
This has been addressed in PR #1071.
